apiary.apib

```diff
@@ -100,6 +112,8 @@ Get menu for customer. If no attributes are provided menu is based on customer d
 
 ### Create direct order [POST /order/direct]
 
+!! Proč se to liší od toho, co chceme (a potřebujeme). Tedy zákazníka, datum, id jídla a množství
```

Comment:
Zákazník je v headeru. Mimo datumu, id jídla a množství se bude posílat ještě vypočítaná cena objednávky pro kontrolu a id výdejny

`  "price": 300,
  "item": {
    "menuItemId": 123,
    "quantity": 2
  },
  "orderDate": "dd-MM-yyyy",
  "canteenId": 12,`

Reply:
OK, dejme tomu. Tak to ale správně popište v tom API, tedy že cena je jenom po porovnání s cenou kterou si stanoví Septim, aby jsme zákazníkovi neukazovali jinou cenu než mu naúčtujeme.

Reply:
Doplněno